stablehlo/reference/Element.cpp

```diff
@@ -263,7 +263,12 @@ std::complex<APFloat> Element::getComplexValue() const {
   return std::complex<APFloat>(floatPair.first, floatPair.second);
 }
 
-bool Element::operator!=(const Element &other) const {
+Element Element::operator!() const {
+  return Element(mlir::Builder(this->getType().getContext()).getI1Type(),
+                 !this->getBooleanValue());
```

Comment:
Let's use `IntegerType::get(getType().getContext(), 1)` directly instead, to avoid creating builders?